dashboard/app/helpers/restricted_markdown_helper.rb

```diff
@@ -0,0 +1,38 @@
+require 'redcarpet'
+require 'redcarpet/render_strip'
+
+# A markdown renderer that strips everything, except for codespans (via backticks)
+module RestrictedMarkdownHelper
```

Comment:
Unless you want to keep the name generic to accommodate future spec changes, in which case I think as-is is fine.